CIP-????/cddl/version_1.cddl

```diff
@@ -1,8 +1,7 @@
-; "????" to be replaced by the CIP number, as a `uint`.
-metadata = { ???? => action }
+metadata = { "86" => action }
 
 ; =========================================================================
-; CIP-???? actions
+; CIP-86 actions
```

Comment:
Shouldn't it be `0086` as below
![image](https://user-images.githubusercontent.com/28530747/221616601-30d26760-090e-4866-aaed-fcee41d1bc56.png)

Reply:
I'm not sure. People seem content to omit the leading zeroes when e.g. mentioning CIP-25...